util/logtools/encounter_tools.ts

```diff
@@ -24,6 +27,29 @@ type FightEncInfo = ZoneEncInfo & {
   sealName?: string;
 };
 
+const fixedTable = (data: { [index: number]: { [index: string]: string } }): void => {
+  // Original console output interception and rewriting technique by Brickshot
+  // See https://stackoverflow.com/a/69874540
+  const ts = new Transform({
+    transform(chunk, _enc, cb) {
+      cb(null, chunk);
+    }
+  });
+  const logger = new Console({ stdout: ts });
+  logger.table(data);
+  const tsRead = ts.read() as Buffer;
+  let result = '';
+  for (const row of tsRead.toString().split(/[\r\n]+/)) {
+    let r = row.replace(/[^┬]*┬/, '┌');
+    r = r.replace(/^├─*┼/, '├');
+    r = r.replace(/│[^│]*/, '');
+    r = r.replace(/^└─*┴/, '└');
+    r = r.replace(/(\s['"]|['"]\s)/g, '  ');
```

Comment:
Hmm, I don't think this will work for trailing apostrophes, e.g. "Hells' Lid".  What about `replace(/['"](\s+|)/, ' $1')` and `replace(/(|\s+)['"]/, '$1 ')`

Reply:
Very glad regex101 exists, I wouldn't have figured out the Unicode issue with this otherwise!